CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 14.1.0 - [#528](https://github.com/openfisca/openfisca-core/pull/528)
+
+#### New features
+
+- Introduce `/calculate` route in the preview API
+  - Allows to run calculations
+
+- Handle `500` errors in the preview API
+    - In this case, the API returns a JSON with details about the error.
+
+- Allows simulations to be build from a JSON using their constructor
+  - For instance `Simulation(simulation_json = {"persons": {...}, "households": {...}}, tax_benefit_system = tax_benefit_system)`
+
+- Allows entities to be built from a JSON using their constructor
+  - For instance `Household(simulation, {"first_household": {...}})`
```

Comment:
A keyword argument is not need in this case, as `entities_json` is the second argument.

The idea is:
  - There was a `Household(simulation)` constructor
  - Now there is also a `Household(simulation, entities_json)` constructor